### ISSUE TITLE
Evaluation metric for generated text

### ISSUE THREAD

**Issue by julfr** [stefan-it/german-gpt2#14]
Hello, thank you for your great project!
I tried to fine-tune the model with my german text and noticed, that generated sentences are mostly copied from my training dataset.
Could you please advice which metric to use to evaluate the generated sentences? Is there a possibility to check, how much of the generated sentence was "copied" from dataset? Thanks in advance!
